pkg/infra/filestorage/filestorage.go

```diff
@@ -103,58 +84,143 @@ func removeStoragePrefix(path string) string {
 	return strings.Join(split, Delimiter)
 }
 
+func (b service) addBackend(backend backendConfig) error {
+	if backend.Type != BackendTypeFS {
+		// TODO add support for DB
+		return nil
+	}
+
+	if backend.FSBackendConfig == nil {
+		return errors.New("Invalid backend configuration " + backend.Name)
+	}
+
+	fsBackendLogger := log.New("fileStorage-" + backend.Name)
+	path := fmt.Sprintf("file://%s", backend.FSBackendConfig.RootPath)
+	bucket, err := blob.OpenBucket(context.Background(), path)
+	if err != nil {
+		return err
+	}
+
+	// TODO mutex
+	if _, ok := b.backendByName[backend.Name]; ok {
+		return errors.New("Duplicate backend name " + backend.Name)
+	}
+
+	pathFilters := &PathFilters{allowedPrefixes: backend.AllowedPrefixes}
+	b.backendByName[backend.Name] = NewCdkBlobStorage(fsBackendLogger, bucket, "", pathFilters, backend.SupportedOperations)
+	return nil
+}
+
+func (b service) validatePath(path string) error {
```

Comment:
A bit unrelated to these changes, but would be nice with some tests of the path validation since security related. Ref #46089